src/main/java/seedu/duke/tasks/DeadlineTask.java

```diff
@@ -0,0 +1,39 @@
+package seedu.duke.tasks;
+
+import java.time.LocalDateTime;
+
+public class DeadlineTask extends Task {
+    private static final long serialVersionUID = -2528903469387431L;
+
+    private final LocalDateTime deadline;
+
+    /**
+     * Create a deadline with a deadline.
+     *
+     * @param description the deadline description
+     * @param deadline the deadline
+     */
+    public DeadlineTask(String description, LocalDateTime deadline) {
+        super(description);
+        this.deadline = deadline;
+    }
+
+    /**
+     * Get the deadline's deadline.
+     *
```

Comment:
I think for method with little code (one line) inside, probably may not necessary to add the javadoc?